osquery/filesystem/linux/proc.h

```diff
@@ -59,7 +59,7 @@ const std::vector<std::string> tcp_states = {"UNKNOWN",
                                              "FIN_WAIT1",
                                              "FIN_WAIT2",
                                              "TIME_WAIT",
-                                             "CLOSE",
+                                             "CLOSED",
```

Comment:
@obelisk @muffins thought about renaming this? Its name should be "CLOSED" according to RFC793, netstat output, etc

Since the TCP state is now available on more than one platform it makes sense for them to have the same name. This will break any script on user end however.

Reply:
Yeah that's a good call. We'll tag this with API change and that should be sufficient for giving folks notice, as we'll call it out in our release notes.